_posts/2022-02-23-roadmap-proposal.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Update: OpenSearch Proposed 2022 Release Schedule"
+authors:
+  - henkle
+date: 2022-02-23
+categories:
+  - partners
+twittercard:
+  description: "As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far, and hear your thoughts."
+excerpt: "As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far, and hear your thoughts."
+
+---
+
+As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far. 
+
+### Considerations
+
+Before getting to the dates themselves, here are the things I was thinking about when I picked them. 
+
+1. **The OpenSearch Project uses SemVer.** 
+As a reminder, the OpenSearch project assigns version numbers to releases following [the semantic versioning convention](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/).  This means the project won’t introduce an incompatible change without also incrementing the major version number.  
+
+2.  **Everyone is excited about Lucene 9.0.0.**
+Lucene 9.0.0 [released](https://lucene.apache.org/core/corenews.html#apache-lucenetm-900-available) on December 7, 2021. It includes several new features and performance improvements that the team would like to make available to users of OpenSearch, including K-NN support, Vectors, Big Endian, Faster numeric indexing, faster sorting, concurrent merge scheduler, and prototype Java Jigsaw module support. It will take a few releases for OpenSearch to take full advantage of it, but the potential of Lucene 9.0.0 is so exciting getting support in as soon as possible is the priority over trying to bundle it with other changes.
+
```

Comment:
"It will take a few releases for OpenSearch to take full advantage of it, but the potential of Lucene 9.0.0 is so exciting that we want to start taking advantage of it as soon as possible."
 
I know the word "we" give you hives, but trying to write around it makes this particular sentence feel awkward.

Reply:
The ambiguous we is never the answer ;) FWIW, I didn't care about this before it was surfaced by the community and now I totally see how throwing 'we' around makes the writer look shifty. It's really not a great look, especially if it's inconsistent. "Advantage" twice in the same sentence also gives me hives. 

Two options:

"It will take a few releases for OpenSearch to fully leverage it, but the potential of Lucene 9.0.0 is so exciting that the team wants to start taking advantage of it as soon as possible."

Or

"Starting to use Lucene 9.0.0 as soon as possible is a priority. It will take a few releases to leverage the full value of it, but adding it now is so exciting and has loads of potential for OpenSearch. "

Reply:
Okay, someday you and I are gonna have a Whorfian thumb wrestling match ;)

Either is fine.